### DISCUSSION TITLE
Running super slow today?

### DISCUSSION THREAD

**Discussion by j-ktz** [UltimaHoarder/UltimaScraper#871] [Q&A] [answered]
It usually works great for me but today it's taking foreverrr to do anything. It'll take a few minutes to auth (no info has changed) and then another 2-3 minutes to load my subscriptions (it's only 11, not a long list), then after that it just says missing posts and keeps retrying but nothing downloads. My download speed is what it always is, no VPN, and the website seems to be working fine. Not sure what's up. Anyone else having issues?

**Comment by UltimaHoarder** [comment, marked as answer]
Yes, it's OF's side.
